Source/Fuse.Common/Tests/FuseTest/ObservableList.uno

```diff
@@ -10,7 +10,7 @@ namespace FuseTest
 		
 		NOTE: Taken from the charting in premiumlibs. Stuck in FuseTest for now since there is no use in fuselibs yet other than for testing.
 	*/
-	public abstract class ObservableData : IObservable
+	abstract class ObservableData : IObservable
```

Comment:
Why have these test classes been changed to `internal`?

Reply:
Otherwise these classes expose the internal class IObserver publicly, which will become a compilation error soon.

Reply:
They don't expose it. The implementation methods are private. It's valid to implement an internal internal interface so long as you don't make public implementation functions. C# supports implementing internal interfaces, and we depend on this in many places.

Reply:
Sure, but e.g. this line basically means it's exposed publicly if the surrounding class is public.

```
virtual protected void OnSubscribe(IObserver observer)
```

Reply:
I think you can just make that function, `OnSubscribe` internal. It's only used within that package.

These classes in test are supposed to be public, to simplify testing, otherwise we'll end up with a bunch of InternalsVisibleTo.  If they use an internal method that's okay, but that bit wasn't meant to be leaking.

Reply:
Yeah, that's better. Fixed!